### PR TITLE
Yet another set of links.

### PR DESCRIPTION
**Pull request by gramian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ A curated list of awesome MATLAB toolboxes, applications, software and resources
 * [saveFigure](https://github.com/djoshea/matlab-save-figure) - exports MATLAB® figures to PDF, SVG, EPS and PNG with improved support for transparancy.
 * [hq-matlab-figs](http://dgleich.github.io/hq-matlab-figs/) - Creating high-quality graphics in MATLAB for papers and presentations.
 * [antijet](https://git.io/antijet) - An alternative to the problematic "jet"-colormap.
+* [octave kernel](https://github.com/calysto/octave_kernel) - to create IPython / Jupyter notebooks.
```
Comment (egeerardyn) [inline README.md]: 1. Is there a reason why you include the Octave kernel and not the [MATLAB one](https://github.com/calysto/matlab_kernel)?
2. Also, it is not really graphics-related. I would rather classify this as "interfacing with other languages".
3. I think the description is not correct. Those do not allow to create such notebooks. They allow to run code in Octave/MATLAB from within Jupyter (previously IPython) notebooks and allow to transfer data and graphics between Jupyter and MATLAB/Octave.

File: README.md [hunk 2]
```diff
@@ -168,7 +171,7 @@ Where to discover new MATLAB resources.
 * [MATLAB Reddit](https://www.reddit.com/r/matlab/)
 * [Octave Reddit](https://www.reddit.com/r/octave/)
 * [MATLAB on Facebook](https://www.facebook.com/MATLAB) - Official Facebook page for MATLAB
-* [MATLAB on Stack Overflow](http://stackoverflow.com/questions/tagged/matlab) - Stack Overflow is a community of 4.7 million programmers, just like you, helping each other
+* [MATLAB on Stack Overflow](http://stackoverflow.com/questions/tagged/matlab) - Stack Overflow is a community of 4.7 million programmers, just like you, helping each other (See also [Octave on Stack Overflow](https://stackoverflow.com/questions/tagged/octave))
```
Comment (egeerardyn) [inline README.md]: Perhaps it's clearer to just have the Octave tag as a separate item?

File: README.md [hunk 3]
```diff
@@ -189,6 +192,7 @@ Where to discover new MATLAB resources.
 * [MATLAB Style Guide 1](http://note.sonots.com/Matlab/MatlabCodingStyle.html)
 * [MATLAB Style Guide 2](http://4dpiecharts.com/matlab-code-style-guide/)
 * [Octave and MATLAB Snippets](https://git.io/mtips) - Collection of code snippets
+* [knkutils](https://github.com/kendrickkay/knkutils) - MATLAB utility functions written by Kendrick Kay
```
Comment (egeerardyn) [inline README.md:197]: Overwhelming collection.
Just out of interest: is there anything special you can recommend from this?

**Comment by egeerardyn** [comment]
Awesome :+1:

File: README.md (hunk 3 above)
Comment (gramian) [inline README.md:197]: I find "choose" (programming) useful and if I would use MATLAB I would like "vflatten" (Octave has "vec"). I could see a few more inside anonymous functions.

Comment (egeerardyn) [inline README.md:197]: I first thought that those functions were preposterous.
E.g. `vflatten` is literally just

``` matlab
function [v] = vflatten(v)
v = v(:);
```

with documentation.

However, then I thought again, and saw the pure awesomeness this gives for [functional programming](http://blogs.mathworks.com/loren/2013/01/24/introduction-to-functional-programming-with-anonymous-functions-part-2/), e.g. the [FP constructs for MATLAB](http://www.mathworks.com/matlabcentral/fileexchange/39735-functional-programming-constructs).

Together with the sometimes overly rigid syntax of MATLAB (e.g. `sum(ones(3,1,3))(:)` is illegal), that toolbox is a hidden gem. Thank you :+1:

File: README.md [hunk 4]
```diff
@@ -85,6 +85,8 @@ A curated list of awesome MATLAB toolboxes, applications, software and resources
 * [Fortran mex routines](http://uk.mathworks.com/help/matlab/write-fortran-mex-files.html) - Fortran MEX routines allow you to call Fortran code from within MATLAB.
 * [MATLAB Engine for Python](http://uk.mathworks.com/help/matlab/matlab_external/get-started-with-matlab-engine-for-python.html) - The MATLAB Engine for Python provides a Python package named `matlab` that enables you to call MATLAB functions from within Python.
 * [Python](http://uk.mathworks.com/help/matlab/call-python-libraries.html) - How to call Python functions from within MATLAB.
+* [matlab kernel](https://github.com/calysto/matlab_kernel) - to run matlab code inside IPython / Jupyter notebooks.
```
Comment (egeerardyn) [inline README.md:88]: Would you mind adding capital letters where appropriate (start of sentences, "MATLAB", "Octave", ...) ?

**Pull request merged**